fetcher.sh

```diff
@@ -24,8 +24,14 @@ EOF
 if [ "$kernel" = "Linux" ]; then
 	# get distro
 	# name is saved in the $NAME variable
-	. "/etc/os-release"
-
+	if [[ -f "/etc/os-release" ]] then
+		. "/etc/os-release"
+	fi
+	# Artix support --- On artix
+	# /etc/os-release doesnt exist
+	if [[ ! -f "/etc/os-release" ]] then
+		. "/etc/artix-release"
+	fi
```

Comment:
For starters, this is a shell project, you're using bash code.

second, you replace something that's done in 1 line with something that's slower *and* done in 3 lines.

third, 5 lines to support a crappy distro that nobody uses?

If I were making this change, which again, I am 100% against, I would do

```
. /etc/os-release 2>/dev/null
. /etc/artix-release 2>/dev/null
```